app/models/ext_management_system.rb

```diff
@@ -622,11 +622,33 @@ def destroy_queue
     task.id
   end
 
+  def ems_workers
+    MiqWorker.find_alive.where(:queue_name => queue_name)
+  end
+
+  def wait_for_ems_workers_removal
+    return if Rails.env.test?
+
+    quiesce_loop_timeout = ::Settings.server.worker_monitor.quiesce_loop_timeout || 5.minutes
+    worker_monitor_poll  = (::Settings.server.worker_monitor.poll || 1.second).to_i_with_method
```

Comment:
I grabbed these values from the [worker quiesce code](https://github.com/ManageIQ/manageiq/blob/76aefb0bee8c834a6d26c609d66e87599adb0f78/app/models/miq_server/worker_management/monitor/quiesce.rb#L33-L34).